processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessingContextImpl.java

```diff
@@ -73,6 +77,11 @@ public String key() {
         return request.key();
     }
 
+    @Override
+    public Headers headers() {
+        return headers;
```

Comment:
Any points to add a `field`? Like other getters just returning `request.headers()`  works?

Reply:
Oh nice catch, completely right.